agency/src/services/index.ts

```diff
@@ -13,12 +14,13 @@ import webHooks from './webhooks/webhooks.service';
 
 export function services(app: Application): void {
   app.configure(admin);
+  app.configure(credential);
+  app.configure(events);
```

Comment:
You may want to move this into `internalServices` instead: not a huge difference, but it keeps the contexts well defined and separate. Additionally, all the internal services will NOT have a swagger UI regardless of how they are defined.